DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/README.md

```diff
@@ -214,3 +216,64 @@ The images below show the dataflow in the MVDR beamforming design. The first ima
 <img src="processing_kernels_ideal.png" alt="processing_kernels_ideal" width="800"/>
 <img src="processing_kernels_fake.png" alt="processing_kernels_fake" width="800"/>
 
+### Using Real IO-pipes
+This section describes how to build and run this reference design on a BSP with real IO pipes. The real IO pipes version does **not** work on Windows and requires a specific system setup and BSP.
```

Comment:
Doesn't it only not work on Windows because our BSP does not work on windows?  I think I would specify that.  Also maybe a bit more detail on the system setup.  How about this: 

The real IO pipes version requires a specific system setup and BSP.  The setup involves a physical network connection from a free Ethernet port in your system to the FPGA board.  Note that the BSP targeted in the Real IO-pipes design does not support Windows.

Should we add something about developers being free to target their own BSPs (which might support windows) with minor modifications?  And maybe mention that the connection to the pipes is the only thing that needs to be modified?

Reply:
no. The UDP code for sending data over ethernet is Linux specific